### PR TITLE
ML-137 Add products

### PR DESCRIPTION
**Pull request by kamil-stasiak**

### CODE REVIEW

File: assets/js/components/event/MobileBottomPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  isBannedFromChat,
+}: Props) => {
+  const { device } = useContext(ScreenTypeContext);
+  const animation = selectAnimation(card);
+
+  if (device !== "mobile") return null;
+  return (
+    <div className={`MobileBottomPanel ${animation}`}>
+      <div className="MobileBottomPanel--TopBar">
+        <div className="MobileBottomPanel--TopBar--Bar" onClick={onBarClick} />
```
Comment (Rados13) [inline assets/js/components/event/MobileBottomPanel.tsx:43]: As I don't know much about styling, I have a question whether this `--` in the name stands for something or it is simply convention?

File: assets/js/components/event/MobileRightSidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    {
+      id: "chat",
+      icon: "/icons/comments-regular.svg",
+      text: "CHAT",
+      onClick: () => onButtonClick("chat"),
+    },
+    {
+      id: "products",
+      icon: "/icons/gifts-regular.svg",
```
Comment (Rados13) [inline assets/js/components/event/MobileRightSidebar.tsx:43]: What do you think about moving icons to the directory path to a variable and then using template literals?

File: assets/js/components/event/Sidebar.tsx [hunk 3]
```diff
@@ -154,38 +157,81 @@ const Participant = ({ client, participant, eventChannel }: ParticipantProps) =>
...
+  participants,
+  isBannedFromChat,
+}: ParticipantsListProps) => {
+  const [listMode, setListMode] = useState<SidebarMode>("products");
+  const isMutating = useIsProductMutating();
+
+  const productsMap: Record<string, Product[]> = groupBy(products, (product) => product.id);
+
+  const allProductsQuery = useAllProductsQuery(client.isModerator);
+  const all: Product[] = allProductsQuery?.data?.data?.products || [];
```
Comment (Rados13) [inline assets/js/components/event/Sidebar.tsx:197]: What do you think about renaming this variable to `allProducts`?

**Review by Rados13 — DISMISSED** [review]

File: assets/js/components/event/MobileBottomPanel.tsx (hunk 1 above)
Comment (kamil-stasiak) [reply assets/js/components/event/MobileBottomPanel.tsx]: I removed the `--`

File: assets/js/components/event/StreamArea.tsx [hunk 4]
```diff
@@ -1,53 +1,77 @@
...
+    isBannedFromChat,
+  } = props;
   const [amIPresenter, setAmIPresenter] = useState<boolean>(false);
   const [presenterName, setPresenterName] = useState<string>("");
-  const config = useRef<Partial<HlsConfig>>({
-    liveSyncDurationCount: 2,
-    initialLiveManifestSize: 1,
-    backBufferLength: 30,
-    manifestLoadingMaxRetry: 6,
-  });
```
[the remark this answers is not in the document]
Comment (kamil-stasiak) [reply assets/js/components/event/StreamArea.tsx]: That was a mistake. I reverted it

**Review by Rados13 — APPROVED** [review]

**Pull request merged**